### ISSUE TITLE
Most of the time val_loss is greater than 1! I think it is a big issue for mask-rcnn.

### ISSUE THREAD

**Issue by K-M-Ibrahim-Khalilullah** [matterport/Mask_RCNN#1832]
I am trying to solve over fitting by changing validation and training data. But always val_loss value is greater than 1.  

Is there any specific solution or suggestion please?

@waleedka @moorage

**Comment by b5l** [comment]
@ibrahimLearning Did you find a solution, or did you close due to inactivity? If you found a solution or a direction to look in, could you please share that?

**Comment by kimile599** [comment]
> @ibrahimLearning Did you find a solution, or did you close due to inactivity? If you found a solution or a direction to look in, could you please share that?

I got the same problem. Do you fix it?
